carddrawer/src/test/java/com/meli/android/carddrawer/model/CardUITest.kt

```diff
@@ -18,35 +17,34 @@ class CardUITest: BaseTest() {
     @Before
     override fun setUp() {
         super.setUp()
-        cardUI = mockk {
-            every { bankImageUrl } returns ""
-            every { cardLogoImageUrl } returns ""
-            every { bankImageRes } returns 0
-            every { cardLogoImageRes } returns 0
-            every { securityCodeLocation } returns ""
-            every { cardFontColor } returns 0
-            every { cardBackgroundColor } returns 0
-            every { cardGradientColors } returns mutableListOf()
-            every { securityCodePattern } returns 0
-            every { cardNumberPattern } returns intArrayOf()
-            every { namePlaceHolder } returns ""
-            every { expirationPlaceHolder } returns ""
-            every { fontType } returns ""
-            every { animationType } returns ""
-            every { customFont } returns mockk()
-            every { style } returns mockk()
-            every { disabledColor } returns 0
-        }
```

Comment:
@samantaalbanez I think this sintax is better for understanding that the `every` callls are nested / related to the cardUI mock, what do you think?

Reply:
I changed it because mock wasn't calling the original values ​​and it didn't appear in the test coverage